src/index.js

```diff
@@ -0,0 +1,147 @@
+import _, { add, times } from 'lodash';
+import { clearAll } from './script.js';
+import taskCompleted from './script.js';
+import './style.css';
+
+const myContainer = document.getElementById('to-do-list');
+tasks.forEach((item) => {
+	myContainer.innerHTML += `
+	<ul class='list-container'>
+		<li class='list-items'>
+		<input type="checkbox" id="check">
+		${item.description}
+		</li>
+	</ul>
+`;
+});
+
+myContainer.innerHTML += `
+<div class='footer-list'>
+	<a href='#' class='footer-a'>Clear all completed</a>
+</div>
+`;
+
+
+const formContainer = document.getElementById('form')
+const inputElement = document.getElementById('input-element');
+const myContainer = document.getElementById('items-container');
```

Comment:
- [ ] As mentioned by the previous reviewer, your web app shows an error because you have defined `myContainer` twice, in line 6 and line 27. Please try to fix that.